src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectAcceleratorDetailsController.kt

```diff
@@ -110,8 +111,10 @@ data class ProjectAcceleratorDetailsPayload(
     val perHectareBudget: BigDecimal?,
     val pipeline: Pipeline?,
     val projectArea: BigDecimal?,
+    val projectHighlightPhotoValueId: VariableValueId?,
     val projectId: ProjectId,
     val projectLead: String?,
+    val projectZoneFigureValueId: VariableValueId?,
```

Comment:
"Zone figure" is what we're calling the map?

Reply:
Correct ya, the variable name (stable id 525) is "Project Zone Figure"

Reply:
But also, just because that's what the DB variable's name is, that doesn't mean we should leave our kotlin variable named that. Would you prefer a different name?

Reply:
What are we calling it on the project profile page? I would have thought it'd be called "map image" or something like that, but I assume there's a reason they used the term "zone figure" for the variable name.

I'm actually fine with keeping this name, though. The more consistent we are about naming across different ways of accessing a source of truth, the less confusing things will be.

Reply:
There's currently no title for either picture, it's just a highlight photo and a map photo.
Sounds good, I'll call this one done, and will have one more PR on this stack. Thanks!